client/src/components/AverageRating.jsx

```diff
@@ -4,6 +4,8 @@ import axios from 'axios';
 import Rating from '@material-ui/lab/Rating';
 
 const getRatingInfo = (ratings) => {
+  if (ratings.length === 0) return 0;
```

Comment:
Is it right to return 0 or should you be returning an object with average: 0 and total: 0?

Reply:
I've removed that logic, it is incorrect